content/guides/trash/permanently-delete-a-file.md

```diff
@@ -0,0 +1,19 @@
+---
+rank: 2
+related_endpoints:
+ - delete_files_id
+related_guides: []
```

Comment:
I'd say the restore * guides for all these "permanently" delete guides would be related guides.

Reply:
So in this case:

```
related_guides:
  - trash/restore-a-file
```